internal/backend/sessionsgrpcsvc/svc.go

```diff
@@ -102,15 +102,25 @@ func (s *server) GetLog(ctx context.Context, req *connect.Request[sessionsv1.Get
 		return nil, sdkerrors.AsConnectError(err)
 	}
 
-	hist, err := s.sessions.GetLog(ctx, sessionID)
+	filter := sdkservices.ListSessionLogRecordsFilter{
```

Comment:
Shouldn't the implementation of paging here in `GetLog` be the same as in `List`? Both in terms in input params, and backend handling of them.

BTW, regarding the existing pagination in `List` (not necessarily related to this PR):  if the server can quietly override the user's page size (which is good and important): shouldn't the response also include the actual page size?